From 52e2d75c2c8e32d2e4f69840e34d21b39279284a Mon Sep 17 00:00:00 2001
From: Milind Changire <mchangir@redhat.com>
Date: Thu, 13 Dec 2018 12:46:56 +0530
Subject: [PATCH 33/52] build: add missing explicit package dependencies

Add dependencies for glusterfs-libs, and other packages.
This is an Errata Tool whine.

Label: DOWNSTREAM ONLY

Change-Id: Ieaadb6e4ffa84d1811aa740f7891855568ecbcbb
Signed-off-by: Milind Changire <mchangir@redhat.com>
Reviewed-on: https://code.engineering.redhat.com/gerrit/158501
Tested-by: RHGS Build Bot <nigelb@redhat.com>
Reviewed-by: Sunil Kumar Heggodu Gopala Acharya <sheggodu@redhat.com>
---
 glusterfs.spec.in | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/glusterfs.spec.in b/glusterfs.spec.in
index bdb47ba..9cd4372 100644
--- a/glusterfs.spec.in
+++ b/glusterfs.spec.in
@@ -323,6 +323,7 @@ and client framework.
 Summary:          GlusterFS api library
 Requires:         %{name}%{?_isa} = %{version}-%{release}
 Requires:         %{name}-client-xlators%{?_isa} = %{version}-%{release}
+Requires:         %{name}-libs%{?_isa} = %{version}-%{release}
 
 %description api
 GlusterFS is a distributed file-system capable of scaling to several
@@ -340,6 +341,7 @@ Summary:          Development Libraries
 Requires:         %{name}%{?_isa} = %{version}-%{release}
 Requires:         %{name}-devel%{?_isa} = %{version}-%{release}
 Requires:         libacl-devel
+Requires:         %{name}-api%{?_isa} = %{version}-%{release}
 
 %description api-devel
 GlusterFS is a distributed file-system capable of scaling to several
@@ -391,6 +393,8 @@ Requires:         %{name}%{?_isa} = %{version}-%{release}
 %if ( 0%{!?_without_extra_xlators:1} )
 Requires:         %{name}-extra-xlators%{?_isa} = %{version}-%{release}
 %endif
+Requires:         %{name}-libs%{?_isa} = %{version}-%{release}
+Requires:         %{name}-server%{?_isa} = %{version}-%{release}
 
 %description devel
 GlusterFS is a distributed file-system capable of scaling to several
@@ -435,6 +439,7 @@ Requires:         %{name}-client-xlators%{?_isa} = %{version}-%{release}
 
 Obsoletes:        %{name}-client < %{version}-%{release}
 Provides:         %{name}-client = %{version}-%{release}
+Requires:         %{name}-libs%{?_isa} = %{version}-%{release}
 
 %description fuse
 GlusterFS is a distributed file-system capable of scaling to several
@@ -459,6 +464,7 @@ Requires:         python%{_pythonver}-gluster = %{version}-%{release}
 
 Requires:         rsync
 Requires:         util-linux
+Requires:         %{name}-libs%{?_isa} = %{version}-%{release}
 
 %description geo-replication
 GlusterFS is a distributed file-system capable of scaling to several
@@ -536,6 +542,7 @@ BuildRequires:    libibverbs-devel
 BuildRequires:    librdmacm-devel >= 1.0.15
 %endif
 Requires:         %{name}%{?_isa} = %{version}-%{release}
+Requires:         %{name}-libs%{?_isa} = %{version}-%{release}
 
 %description rdma
 GlusterFS is a distributed file-system capable of scaling to several
@@ -664,6 +671,7 @@ This package provides the glusterfs thin-arbiter translator.
 
 %package client-xlators
 Summary:          GlusterFS client-side translators
+Requires:         %{name}-libs%{?_isa} = %{version}-%{release}
 
 %description client-xlators
 GlusterFS is a distributed file-system capable of scaling to several
-- 
1.8.3.1

